nginx.conf

```diff
@@ -13,13 +13,104 @@
 #
 # One difference between this file and the configuration for production is that
 # this configuration does not support request body decompression.
+daemon off;
+worker_processes 1;
+error_log stderr;
+pid ./tmp/nginx.pid;
 
 events {
 }
 http {
-  # This path may be different on your local computer depending on your OS and
-  # how you installed nginx.
-  include /usr/local/etc/nginx/mime.types;
+  access_log ./tmp/nginx-access.log;
```

Comment:
Similar as above, is this addition needed?

Reply:
Yes, similarly to `error_log`, this may otherwise default to an absolute path which a normal user may not have permissions for, and if they are using nginx for other purposes may pollute their logs.